### PR TITLE
Feat(displaymodes): add priv/sec note (closes #402)

### LINKED ISSUE #402: Privacy Review: how to handle navigation to a fullscreen'd application?
It is actually more of a security issue than a privacy issue. If a user has an application installed and the UA wants them to use it instead of the regular browser experience when clicking on a link, it might be very easy for that application to spoof the user and make them believe they are on another website.

The specification should at least mention that issue and provide some guidance. Maybe show the origin in that case? Always allow the user to find the URL bar?

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by marcoscaceres** [comment]
@kenchris, your magic touch please?

File: index.html [hunk 1]
```diff
@@ -556,6 +556,30 @@ <h3 id="navigation-scope-security-considerations">
       </p>
       <section>
         <h3>
+          Privacy and security considerations
+        </h3>
+        <p>
+          When the <a>fullscreen</a> <a>display mode</a> is applied, it is
+          RECOMMENDED that the user agent still afford the end-user a means to
```
Comment (kenchris) [inline index.html]: it afford*s* but I am not sure afford is the right word here. Why not use 'allows'

File: index.html [hunk 2]
```diff
@@ -556,6 +556,30 @@ <h3 id="navigation-scope-security-considerations">
...
+          access/view the origin, or URL, and icon of the Web application. This
```
Comment (kenchris) [inline index.html]: ... RECOMMENDED that the user agent provides the end-user a means to consult basic information about the web apps, such as the origin, complete URL, granted permissions and associated icon.

Such information shouldn't necessarily need to overlay the application itself, but can instead be shown in the task switcher, for instance by clicking on an info button.

Comment (kenchris) [inline index.html]: ... RECOMMENDED that the user agent provides the end-user a means to consult basic information about the web apps, such as the origin, complete URL, granted permissions and associated icon.

Such information shouldn't necessarily need to overlay the application itself, but can instead be shown in the task switcher, for instance by clicking on an info button.

Comment (kenchris) [inline index.html]: ... RECOMMENDED that the user agent provides the end-user a means to consult basic information about the web apps, such as the origin, complete URL, granted permissions and associated icon.

Such information shouldn't necessarily need to overlay the application itself, but can instead be shown in the task switcher, for instance by clicking on an info button.

Comment (kenchris) [inline index.html]: ... RECOMMENDED that the user agent provides the end-user a means to consult basic information about the web apps, such as the origin, complete URL, granted permissions and associated icon.

Such information shouldn't necessarily need to overlay the application itself, but can instead be shown in the task switcher, for instance by clicking on an info button.

File: index.html [hunk 3]
```diff
@@ -556,6 +556,30 @@ <h3 id="navigation-scope-security-considerations">
...
+          does not necessarily mean overlaying the origin and icon over the
+          application itself. Rather, for example, the origin could be shown in
+          the OS's task manager or some such.
+        </p>
+        <p>
+          Additionally, when applying a manifest that changes the <a>display
```
Comment (kenchris) [inline index.html]: which sets the

File: index.html [hunk 4]
```diff
@@ -556,6 +556,30 @@ <h3 id="navigation-scope-security-considerations">
...
+          mode</a> to anything except "<a>browser</a>", it is RECOMMENDED that
+          the user agent clearly indicate to the end-user that their are
+          leaving the normal browsing context of a web browser. Ideally,
+          launching or switching to a web application is performed in a manner
+          that is consistent with launching or switching to other applications
+          in the OS. For example, a long and obvious animated transition, or
+          speaking the text "Launching application X", or whatever is customary
```
Comment (kenchris) [inline index.html]: I would avoid the word 'whatever' it sounds like you dont give a shi... :)

File: index.html [hunk 5]
```diff
@@ -556,6 +556,30 @@ <h3 id="navigation-scope-security-considerations">
...
+          on the host <abbr>OS</abbr>.
```
Comment (kenchris) [inline index.html]: Why not say platform instead of OS

File: index.html (hunk 3 above)
Comment (kenchris) [inline index.html]: which sets the

File: index.html (hunk 4 above)
Comment (kenchris) [inline index.html]: I would avoid the word 'whatever' it sounds like you dont give a shi... :)

File: index.html (hunk 5 above)
Comment (kenchris) [inline index.html]: Why not say platform instead of OS

File: index.html (hunk 1 above)
Comment (marcoscaceres) [inline index.html]: Argh, now I can't sound like a UX hipster :(

File: index.html (hunk 4 above)
Comment (marcoscaceres) [inline index.html]: But I don't :) Fixing.

File: index.html (hunk 5 above)
Comment (marcoscaceres) [inline index.html]: Used platform.

File: index.html (hunk 2 above)
Comment (marcoscaceres) [inline index.html]: This presupposes access to the OS's task switcher - which, for instance, wouldn't be true for Firefox on Android. I'm just going to say that it's left up to the implementer to come up with something.

**Pull request merged**